### PR TITLE
Implement Cairo0 CSL examples

### LINKED ISSUE #602: Implement CSL-Examples in CairoZero
- [x] Implement, prove, and verify Transfer
- [x] Implement, prove, and verify Svm5
- [x] Implement, prove, and verify Transfer5000
- [x] Implement, prove, and verify Perceptron

- [x] Measure the times with a bash script

### PR DESCRIPTION
**Pull request by yzhang90**
Closes #602

### CODE REVIEW

File: cairo/csl-examples/cairo0/bench.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    echo "Execution time" ".." $(("$TOTAL" / 1000)).$(("$TOTAL" % 1000)) "s" >> "$PWD/$1"
+
+    # Prove
+    echo "Prove .."  >> "$PWD/$1"
+    platinum-prover prove ${BUILDDIR}/${filename}/${filename}_trace.json ${BUILDDIR}/${filename}/${filename}_memory.json ${BUILDDIR}/${filename}/${filename}.proof >> "$PWD/$1"
+
+    # Verify
+    echo "Verify .." >> "$PWD/$1"
+    platinum-prover verify ${BUILDDIR}/${filename}/${filename}.proof >> "$PWD/$1"
+
```
Comment (Robertorosmaninho) [inline cairo/csl-examples/cairo0/bench.sh:43]: We're not measuring the time of `Prove` and `Verify` here, I assume we're getting the time from the platinum-prover output and passing it for the file.

Isn't the file getting too big and confusing? Shouldn't we, at least, filter only the info that we want? Usually, we would like only one line per test.

File: Performance.md [hunk 2]
```diff
@@ -28,6 +28,16 @@
...
 * Using `lurk --rc 400 transfer5000.lurk`, other tests doesn't use `--rc`
 
+
+## Cairo0
+|     Examples     | CPU Exec Time | CPU Prove Time | CPU Verify Time | CPU Total Time |
+|:-----------------|:-------------:|:--------------:|:---------------:|:---------------|
+| perceptron       |         0.438 |          0.149 |           0.007 |          0.594 |
+| svm5             |         0.442 |          0.175 |           0.007 |          0.624 |
+| transfer         |         0.441 |          0.185 |           0.008 |          0.634 |
+| transfer5000*    |         6.799 |         26.311 |           0.867 |         33.977 |
```
Comment (Robertorosmaninho) [inline Performance.md:38]: Great numbers! Could you provide an example to show me which output number you take to insert here?

File: cairo/csl-examples/cairo0/bench.sh (hunk 1 above)
Comment (yzhang90) [reply cairo/csl-examples/cairo0/bench.sh]: done

**Review by Robertorosmaninho — APPROVED** [review]
LGTM

**Pull request merged**